jxl/src/render/low_memory_pipeline/save/mod.rs

```diff
@@ -67,16 +68,22 @@ impl SaveStage {
 
         let save_size = (save_end.0 - save_start.0, save_end.1 - save_start.1);
 
-        let num_fast = match self.orientation {
-            Orientation::Identity => identity::store(
-                data,
-                frame_y,
-                save_start.0..save_end.0,
-                buf,
-                relative_y,
-                self.data_format,
-            ),
-            _ => 0,
+        // Fast path for identity orientation when not premultiplying
```

Comment:
I don't think this is the correct way of handling this.
Instead, we should create a new stage that premultiplies alpha into color channels, and gets inserted just before the save stages.